drivers/virtualbox/virtualbox.go

```diff
@@ -205,7 +205,10 @@ func cmdOutput(name string, args ...string) (string, error) {
 
 // IsVTXDisabledInTheVM checks if VT-X is disabled in the started vm.
 func (d *Driver) IsVTXDisabledInTheVM() (bool, error) {
-	file, err := os.Open(path.Join(d.ResolveStorePath(d.MachineName), "Logs", "VBox.log"))
+	logPath := path.Join(d.ResolveStorePath(d.MachineName), "Logs", "VBox.log")
+	log.Debugf("Checking vm logs: %s", logPath)
+
+	file, err := os.Open(logPath)
```

Comment:
Looks like you may be missing a `defer file.Close()` after this `err` check block?